src/clusterfuzz/_internal/tests/core/bot/tasks/utasks/uworker_io_test.py

```diff
@@ -0,0 +1,287 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for uworker_io."""
+
+import os
+import shutil
+import tempfile
+import unittest
+from unittest import mock
+
+from google.cloud import ndb
+
+from clusterfuzz._internal.bot.tasks.utasks import uworker_io
+from clusterfuzz._internal.datastore import data_types
+from clusterfuzz._internal.tests.test_libs import helpers
+from clusterfuzz._internal.tests.test_libs import test_utils
+
+DEFAULT_SIGNED_URL_MINUTES = 1440
```

Comment:
nit: `DEFAULT_SIGNED_URL_MINUTES = 24 * 60`.
I recall seeing you wrote something similar in any file, and I thought that made it clearer.